src/diffusers/models/unets/unet_motion_model.py

```diff
@@ -304,6 +370,20 @@ def __init__(
         if isinstance(transformer_layers_per_block, int):
             transformer_layers_per_block = [transformer_layers_per_block] * len(down_block_types)
 
+        if isinstance(reverse_transformer_layers_per_block, int):
```

Comment:
I think we can use a similar approach to:
https://github.com/huggingface/diffusers/blob/2f6f426f669ec0ba715ddb941121198ec6ab11d0/src/diffusers/models/unets/unet_2d_condition.py#L415-L419

when setting the reversed blocks if the argument is provided